oas.go

```diff
@@ -172,6 +172,7 @@ type SchemaObject struct {
 	Items                *SchemaObject          `json:"items,omitempty"` // use ptr to prevent recursive error
 	Example              interface{}            `json:"example,omitempty"`
 	Deprecated           bool                   `json:"deprecated,omitempty"`
+	Enum                 []string               `json:"enum,omitempty"`
```

Comment:
OpenAPI enums don't have to be strings, though I guess for our use case at least we're unlikely to need to support anything else.

Reply:
good point I did not think about further support since all of ours is currently strings. We may want to backlog a ticket for additional support in the future.